src/lang/english.txt

```diff
@@ -2425,25 +2427,25 @@ STR_BRIDGE_TUBULAR_SILICON                                      :Tubular, Silico
+STR_ROAD_TOOLBAR_TOOLTIP_BUILD_TRUCK_LOADING_BAY                :{BLACK}Build Lorry Loading Bay{NBSP}-{NBSP}{}Ctrl+Click enables merging stations. Shift+Click shows estimated cost without purchase
+STR_ROAD_TOOLBAR_TOOLTIP_BUILD_CARGO_TRAM_STATION               :{BLACK}Build Freight Tram Station{NBSP}-{NBSP}{}For loading and unloading cargo. Ctrl+Click enables merging stations. Shift+Click shows estimated cost without purchase
 STR_ROAD_TOOLBAR_TOOLTIP_TOGGLE_ONE_WAY_ROAD                    :{BLACK}Activate/Deactivate one way roads
-STR_ROAD_TOOLBAR_TOOLTIP_BUILD_ROAD_BRIDGE                      :{BLACK}Build road bridge. Shift toggles building/showing cost estimate
-STR_ROAD_TOOLBAR_TOOLTIP_BUILD_TRAMWAY_BRIDGE                   :{BLACK}Build tramway bridge. Shift toggles building/showing cost estimate
-STR_ROAD_TOOLBAR_TOOLTIP_BUILD_ROAD_TUNNEL                      :{BLACK}Build road tunnel. Shift toggles building/showing cost estimate
-STR_ROAD_TOOLBAR_TOOLTIP_BUILD_TRAMWAY_TUNNEL                   :{BLACK}Build tramway tunnel. Shift toggles building/showing cost estimate
-STR_ROAD_TOOLBAR_TOOLTIP_TOGGLE_BUILD_REMOVE_FOR_ROAD           :{BLACK}Toggle build/remove for road construction
-STR_ROAD_TOOLBAR_TOOLTIP_TOGGLE_BUILD_REMOVE_FOR_TRAMWAYS       :{BLACK}Toggle build/remove for tramway construction
-STR_ROAD_TOOLBAR_TOOLTIP_CONVERT_ROAD                           :{BLACK}Convert/Upgrade the type of road. Shift toggles building/showing cost estimate
-STR_ROAD_TOOLBAR_TOOLTIP_CONVERT_TRAM                           :{BLACK}Convert/Upgrade the type of tram. Shift toggles building/showing cost estimate
+STR_ROAD_TOOLBAR_TOOLTIP_BUILD_ROAD_BRIDGE                      :{BLACK}Build Road Bridge{NBSP}-{NBSP}{}Places a road bridge over land or water. Shift+Click shows estimated cost without purchase
+STR_ROAD_TOOLBAR_TOOLTIP_BUILD_TRAMWAY_BRIDGE                   :{BLACK}Build Tramway Bridge{NBSP}-{NBSP}{}Places a tramway bridge over land or water. Shift+Click shows estimated cost without purchase
+STR_ROAD_TOOLBAR_TOOLTIP_BUILD_ROAD_TUNNEL                      :{BLACK}Build Road Tunnel{NBSP}-{NBSP}{}Underground passage for road vehicles. Shift+Click shows estimated cost without purchase
+STR_ROAD_TOOLBAR_TOOLTIP_BUILD_TRAMWAY_TUNNEL                   :{BLACK}Build Tramway Tunnel{NBSP}-{NBSP}{}Underground passage for tramway vehicles. Shift+Click shows estimated cost without purchase
+STR_ROAD_TOOLBAR_TOOLTIP_TOGGLE_BUILD_REMOVE_FOR_ROAD           :{BLACK}Toggle Remove Road Construction{NBSP}-{NBSP}{}Clears roads and stations in a selected area
+STR_ROAD_TOOLBAR_TOOLTIP_TOGGLE_BUILD_REMOVE_FOR_TRAMWAYS       :{BLACK}Toggle Remove Tramway Construction{NBSP}-{NBSP}{}Clears tramways and stations in a selected area
```

Comment:
This seems inaccurate. It does not remove both road/tram and stations, and not in an area. It removes the type and direction of construction from selected tiles, and touches nothing else. I'm not sure if that's the best way to describe it concisely.

Reply:
I'll try to re-word it.

Reply:
Reworded now!